nipype/algorithms/tests/test_auto_CompCor.py

```diff
@@ -1,37 +0,0 @@
-# AUTO-GENERATED by tools/checkspecs.py - DO NOT EDIT
-from __future__ import unicode_literals
-from ..confounds import CompCor
-
-
-def test_CompCor_inputs():
```

Comment:
because autotests are only added for an interface if a named test doesn't exist. and we have `test_compcor.py`